dev-client/src/screens/ProjectListScreen.tsx

```diff
@@ -86,23 +99,61 @@ export const ProjectListScreen = () => {
         )}
 
         {activeProjects.length > 0 && (
-          <>
-            <SearchBar
-              query={query}
-              setQuery={setQuery}
-              placeholder={t('projects.search.placeholder')}
-              FilterOptions={<Text>Project filter placeholder</Text>}
-            />
-            <FlatList
-              data={searchedProjects}
-              renderItem={({item}) => <ProjectPreviewCard project={item} />}
-              ItemSeparatorComponent={() => <Box h="8px" />}
-              keyExtractor={project => project.id}
-              ListEmptyComponent={
-                <Text>{t('projects.search.no_matches')}</Text>
-              }
-            />
-          </>
+          <ListFilterProvider
+            items={activeProjects}
+            filters={{
+              search: {
+                kind: 'filter',
+                f: searchText,
+                preprocess: normalizeText,
+                lookup: {key: 'name'},
+                hide: true,
+              },
+              role: {
+                kind: 'filter',
+                f: equals,
+                lookup: {key: 'id', record: projectRoleLookup},
+              },
+              sort: {
+                kind: 'sorting',
+                options: {
+                  nameAsc: {
+                    key: 'name',
+                    order: 'ascending',
+                  },
+                  nameDesc: {
+                    key: 'name',
+                    order: 'descending',
+                  },
+                },
+              },
+            }}>
+            <ListFilterModal
+              searchInput={
+                <TextInputFilter
+                  name="search"
+                  label={t('projects.search_label')}
+                  placeholder={t('projects.search_placeholder')}
+                />
+              }>
+              <SelectFilter
+                name="sort"
+                label={t('projects.sort_label')}
+                options={SORT_OPTIONS}
+              />
+              <SelectFilter
+                name="role"
+                label={t('projects.role_filter_label')}
+                placeholder=""
```

Comment:
question: if this is empty, can `placeholder=""` be omitted?

Reply:
This changes in the next PR I have prepared to be optional, but I would have to change code that I have already changed there to get this to work here. So I don't think it makes sense to change it here.